baseplate/context/memcache/lib.py

```diff
@@ -0,0 +1,108 @@
+from six import string_types
+import logging
+import zlib
+
+
+try:
+    import cPickle as pickle
```

Comment:
you can grab all these imports safely from the compat module without try/except shenanigans.